ReactiveCocoa/AppKit/NSControl.swift

```diff
@@ -96,7 +96,7 @@ extension Reactive where Base: NSControl {
 		return associatedValue { base in
 			let (signal, observer) = Signal<(), NoError>.pipe()
 
-			let receiver = CocoaTarget(observer)
+			let receiver = CocoaTarget<()>(observer)
```

Comment:
Sorry for a late nit: I think the type inferrer should be able to infer it. Can you try to remove `<()>`? If it works, please do the same to the one in UIControl too. Thanks.